site/app/libraries/ApiRouter.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
let's make a new folder `site/app/libraries/routers` and move all three routers under this folder (rename Router.php to ClassicRouter.php)